src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int MAX_RANDOM_VALUE = 10;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private int getRandomValue() {
+        return random.nextInt(MAX_RANDOM_VALUE) + 1;
+    }
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+        /*I'm sorry, I have a question.
+        Maybe I should use getRandomValues() when creating a new object?*/
+        switch (random.nextInt(FIGURE_NUMBER)) {
```

Comment:
you create array with 5 figures just to pick one of them. Lets use switch-case construction. We can create 5 private methods getRandomCircle(), getRandomSquare(), ... and call them in switch-case construction